yt/utilities/answer_testing/utils.py

```diff
@@ -282,8 +279,92 @@ def _save_arrays(save_dir_name, fbasename, arrays, answer_store):
     answer_store : bool
         If true, save the just-generated test results, otherwise,
         compare them to the previously saved results.
+
+    func_name : str
+        The name of the function that created the answers being saved.
+    """
+    answer_file = os.path.join(save_dir_name, fbasename + '.h5')
+    if answer_store:
```

Comment:
I would suggest changing the name of this variable to `store_answer` to clarify that "store" is meant to read as a verb. In particular since we have a subrepo that's named `answer-store`, I found the current name a bit confusing.